### PR TITLE
Non-CLA submissions

### LINKED ISSUE #9511: Localization error in the header
**Describe the bug**
There are several issues related to Russian translation in the header

**To Reproduce**
1. Go to main page
2. Change locale to russian 
3. Look at the headers

**Expected behavior**
The proper translation would be something like 
"О сайте", "Изучить",  "Учиться", "Паттерны"

**Screenshots**
image

### PR DESCRIPTION
**Pull request by AaronForinton**
https://github.com/GoogleChrome/web.dev/pull/9512
https://github.com/GoogleChrome/web.dev/pull/9228
https://github.com/GoogleChrome/web.dev/pull/9249
https://github.com/GoogleChrome/web.dev/pull/9412
https://github.com/GoogleChrome/web.dev/pull/9257

Fixes #9511

### CODE REVIEW

**Comment by AaronForinton** [comment]
@sofiayem A couple of these were quick fixes on Russian words. Could you please confirm that they're good to go?

File: src/site/content/ru/metrics/lcp/index.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ tags:
 
 {% Img src="image/admin/e0O2woQjZJ92aYlPOJzT.png", alt="Временная шкала LCP с google.com", width="800", height="311" %}
 
-В первом примере логотип Instagram загружается относительно рано и остается самым большим элементом, даже по мере отрисовки другого контента. В примере страницы результатов поиска Google самый большой элементэто абзац текста, который отображается до завершения загрузки любого логотипа или изображения. Так как все отдельные изображения меньше этого абзаца, он остается самым большим элементом на протяжении всего процесса загрузки.
+В первом примере логотип Instagram загружается относительно рано и остается самым большим элементом, даже по мере отрисовки другого контента. В примере страницы результатов поиска Google самый большой элемент это абзац текста, который отображается до завершения загрузки любого логотипа или изображения. Так как все отдельные изображения меньше этого абзаца, он остается самым большим элементом на протяжении всего процесса загрузки.
```
Comment (knenkne) [inline src/site/content/ru/metrics/lcp/index.md:113]:
```suggestion
В первом примере логотип Instagram загружается относительно рано и остается самым большим элементом даже по мере отрисовки другого контента. В примере страницы результатов поиска Google самый большой элемент - это абзац текста, который отображается до завершения загрузки любого логотипа или изображения. Так как все отдельные изображения меньше этого абзаца, он остается самым большим элементом на протяжении всего процесса загрузки.
```

**Comment by knenkne** [comment]
@sofiayem, can u please also review these two fixes https://github.com/GoogleChrome/web.dev/pull/9516 & https://github.com/GoogleChrome/web.dev/pull/9516?

**Review by sofiayem — APPROVED** [review]

**Comment by sofiayem** [comment]
> @sofiayem, can u please also review these two fixes #9516 & #9516?

Hi, you've given two links that are the same. I assume you meant the second to be different. Which one is it?

File: src/site/_data/i18n/header.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ learn:
   ja: 学習する
   ko: 배우기
   pt: Aprender
-  ru: Обучение
+  ru: Учиться
```
Comment (sofiayem) [inline src/site/_data/i18n/header.yml:7]:
```suggestion
  ru: Обучение
```

**Review by sofiayem — APPROVED** [review]

**Comment by knenkne** [comment]
> > @sofiayem, can u please also review these two fixes #9516 & #9516?
> 
> Hi, you've given two links that are the same. I assume you meant the second to be different. Which one is it?

Sorry, another one is https://github.com/GoogleChrome/web.dev/pull/9518

**Pull request merged**